App.tsx

```diff
@@ -1,21 +1,24 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StatusBar } from "expo-status-bar";
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import Homepage from "./components/homepage/homepage.component";
```

Comment:
Can you pull out homepage.component from a folder. I don't see why we would need a folder since the styling is already included in the homepage.component file

Reply:
Well this is just a wireframe, a homepage folder is convenient since there are likely to be other related custom components that are going to be used in the home page screen once the Figma is more flushed out. Not to mention, even if the Home Page itself remains pretty basic and doesn't need any more components, other screens certainly will, so it makes sense to maintain the same folder structure throughout the project. 

I think it's also the recommended way of organizing components (https://www.reactnativeschool.com/how-to-organize-your-components).